zbsprimarystorage/zbsprimarystorage/zbsagent.py

```
@@ -243,13 +236,45 @@ def expand_volume(self, req):
         if ret.error.code != 0:
-            raise Exception('failed to expand lun[%s], error[%s]' % (cmd.lunName, ret.error.message))
+            raise Exception('failed to expand volume[%s], error[%s]' % (cmd.lunName, ret.error.message))
 
         o = zbsutils.query_volume_info(cmd.logicalPoolName, cmd.lunName)
         rsp.size = jsonobject.loads(o).result.info.fileInfo.length
 
         return jsonobject.dumps(rsp)
 
+    @replyerror
+    def flatten_volume(self, req):
+        cmd = jsonobject.loads(req[http.REQUEST_BODY])
```

Ruff (0.11.9)

`http` may be undefined, or defined from star imports

(F405)